alphagov/pay-connector
# PP-6029: Add PaaS manifest

## WHAT YOU DID

This adds a manifest for pushing connector up to PaaS. There are a
number of environment variables that will end up being provided by PaaS
services and the values in `manifest.yml` and `dev.yml` are just
placeholders.

The PaaS Java buildpack doesn't actually build the apps so a JAR has to
be built beforehand. You can point to a custom JAR path with the `-p`
flag to `cf push`.

## How to test

Usage:

    mvn clean package

    cf push --vars-file dev.yml \
      --var card_connector_route=card-connector-<space>.apps.internal \
      --var frontend_url=<frontend url> \
      --var space=<space name>

- Run the command to push connector to your own PaaS developer environment.
- Check that connector starts up and runs successfully.

## Code review checklist

### Logging


### Documentation


* Introduced any new environment variables / removed existing environment variable
* Added new API / updated existing API definition
